scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -2007,8 +2042,6 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
           case p => p.require[Pat]
         }
         makeTuple[Pat](patterns, () => Lit.Unit(), Pat.Tuple(_))
-      case _: Unquote =>
-        unquote[Pat]
```

Comment:
But in Trees we have `class Extract(ref: Term.Ref, targs: Seq[impl.Type], args: Seq[Pat.Arg])`.